HW2/P3/P3.txt

```diff
@@ -0,0 +1,9 @@
+Multithreading
+1 thread -> 1074.656613 Million Complex FMAs in 6.42491793633 seconds, 167.263866037 million Complex FMAs / second
+2 threads -> 1074.656613 Million Complex FMAs in 2.8487610817 seconds, 377.236483574 million Complex FMAs / second
+4 threads -> 1074.656613 Million Complex FMAs in 1.56879615784 seconds, 685.01991647 million Complex FMAs / second
+
+Instruction Level
+1 thread -> 1074.656611 Million Complex FMAs in 0.925685882568 seconds, 1160.93010733 million Complex FMAs / second
+2 threads -> 1074.656611 Million Complex FMAs in 0.413547039032 seconds, 2598.63210124 million Complex FMAs / second
+4 threads -> 1074.656611 Million Complex FMAs in 0.272787094116 seconds, 3939.54345414 million Complex FMAs / second
```

Comment:
You commented below in P4 that your computer has four cores and jumping from two cores to four cores does not increase performance too much because of background processes taking up the core schedules. However, here 4 threads seem to be significant. I guess what you said makes sense as 1 thread to 2 threads performance up by more than 2 times, but 2 threads to 4 threads not that much, but background process might not serve to be the best reason for insignificant performance increase later.